Code/GraphMol/Descriptors/MolSurf.cpp

```diff
@@ -12,6 +12,7 @@
 #include <GraphMol/RDKitBase.h>
 #include <GraphMol/Descriptors/MolDescriptors.h>
 #include <GraphMol/PartialCharges/GasteigerCharges.h>
+#include <GraphMol/Descriptors/MolData3Ddescriptors.h>
```

Comment:
I think retrieving the custom property value is simple enough that you don't need this import and the `MolData3Ddescriptors` object.